x-hooks/use-bounty.tsx

```diff
@@ -142,24 +150,25 @@ export function useBounty() {
         dispatch(changeCurrentBountyDataChain.update(bounty));
 
         state.Service.active.getERC20TokenData(bounty.transactional)
-          .then(token => dispatch(changeCurrentBountyDataTransactional(token)))
+          .then(token => dispatch(changeCurrentBountyDataTransactional(token)));
 
-        if(bounty.rewardToken !== Defaults.nativeZeroAddress)
+        if(bounty.rewardToken && bounty.rewardToken !== Defaults.nativeZeroAddress)
```

Comment:
you can use `isAddress(bounty.rewardToken) && !isZeroAddress(bounty.rewardToken)`